.gitignore

```diff
@@ -21,3 +21,5 @@ android
 # Jetbrains IDE files
 .idea
 
+# Ignore changes to the package-lock
```

Comment:
I added this because I noticed that the `package-lock.json` wasn't being tracked